packages/data/tests/unit/serializers/elide-facts-test.ts

```diff
@@ -0,0 +1,66 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import NaviFactSerializer from 'navi-data/serializers/fact-interface';
+import { AsyncQueryResponse, QueryStatus, QueryResultType, RequestV1 } from 'navi-data/adapters/fact-interface';
+
+const Payload: AsyncQueryResponse = {
+  asyncQuery: {
+    edges: [
+      {
+        node: {
+          id: 'c7d2fe70-b63f-11ea-b45b-bf754c72eca6',
+          query: '"{ "query": "{ starmap_monthly_presto_view { edges { node { datestamp user_count } } } } " }',
```

Comment:
```suggestion
          query: '"{ "query": "{ tableA { edges { node { datestamp user_count } } } } " }',
```

Not sure it matters, but it might be better to use more generic table names